rewards/README.md

```diff
@@ -4,40 +4,80 @@ cover: ../.gitbook/assets/cover/rewards_banner.png
 coverY: 0
 ---
 
-# 💰 Rewards
```

Comment:
My general feeling is that we are giving way too much information on this README. It should be an intro and the rest detailed in the following pages

Reply:
https://github.com/oceanprotocol/docs/issues/1234